frontend/src/components/collapse-component/collapse-element/styles.module.scss

```diff
@@ -15,7 +15,7 @@
 
 .show {
   height: auto;
-  max-height: 1500px;
+  max-height: 2000px;
   transition-duration: 0.8s;
 }
```

Comment:
Why `2000px`?

Reply:
We have dynamic tables height and 1500px was not enough on some screen sizes

Reply:
Okay. Why do we need to have `max-height` at all?

Reply:
That's an interesting question, but collapse component don't work without it